mammoth_gazebo/launch/mammoth.launch.py

```diff
@@ -126,6 +126,29 @@ def generate_launch_description():
         ],
         output='screen'
     )
+    
+    Pointcloud_to_Laserscan = Node(
```

Comment:
dont use capital letters like this for a variable. notice how the others are lower case snake case. this leads people to think that your variable is a proper class or static function